src/test/java/strategy/impl/BalanceOperationImplTest.java

```diff
@@ -0,0 +1,45 @@
+package strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import db.Storage;
+import model.FruitTransaction;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationImplTest {
+    private static final String BANANA = "banana";
+    private static final int OPERATION_AMOUNT = 10;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeClass
+    public static void beforeAll() {
+        Storage.map.clear();
+    }

```

Comment:
You really don't need to do it, It's enough to have only a method with annotation [@After](https://stackoverflow.com/questions/6830342/different-teardown-for-each-test-in-junit) where you clear storage
fix in all places